components/molecules/InsightPageCard/insight-page-card.tsx

```diff
@@ -54,6 +54,17 @@ const InsightPageCard = ({ insight, user }: InsightPageCardProps): JSX.Element =
     },
   ];
 
+  // Function to handle copy to clipboard
+  const handleCopyToClipboard = async (content: any) => {
+    const url = new URL(content, window.location.origin).toString();
+    try {
+      await navigator.clipboard.writeText(url);
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.log(error);
+    }
+  };
+
```

Comment:
I believe this is work that @OgDev-01 was doing. The `handleCopyToClipboard` shouldn't be a part of this PR.

Reply:
@nickytonline so what do you suggest I do with the situation on ground

Reply:
In your branch from the command line, you can run `git checkout upstream/beta -- components/molecules/InsightPageCard/insight-page-card.tsx` or if it says it can't find `upstream` run `git checkout beta -- components/molecules/InsightPageCard/insight-page-card.tsx`.

This will put that file back to what it is on the beta branch.

Reply:
@nickytonline, I ran the line of code in my terminal and I solved the issue by inputting `// eslint-disable-next-line no-console` on line 61 of the `insight-page-car.tsx` file but am still getting a failing code standard. What could be the problem with the project and what am i doing wrongly that is making the eslint fail code standard

Reply:
I've fixed this file by reverting it back to what it is in the `beta` branch. The change is in d2e73a9c29d06c949cc5a81ca82b6d29fbf755cd.

The only file to add the eslint rule is `./lib/utils/copy-to-clipboard.ts`.

After that this should be good.

Reply:
Grab the latest of your branch by running `git pull` when you have a chance and make the suggested change above. Thanks for your patience @brainconnect93!